lib/KSelect/KeenUiSelect.vue

```diff
@@ -552,6 +553,10 @@
           break;
         }
       }
+
+      // if located within KModal, special styles will be applied
+      const kModalCheck = document.getElementsByClassName('modal');
```

Comment:
Could update this check to something like:
```
this.$el.querySelect(".modal .ui-select") === this.$el
```
to be more precise that _this_ ui-select is inside a modal.

Reply:
thanks! i'll look into implementing something along these lines tomorrow morning!

Reply:
@rtibbles i ended up following the spirit of your advice with `querySelectorAll()` after realizing that if there were multiple `KSelect`s located within a single modal, `this.isInsideModal` would only correctly register as `true` for the first `KSelect` because when `querySelector` was called by the other `KSelect`s it would grab the first `KSelect` it found and stop looking, so the comparison would fail and the other `KSelect`s would lack their intended styling.

unfortunately, while checking the intricacies of these components' behavior when a `KModal` contains 2+ `KSelect`s I discovered a recurrence some minor undesirable modal-elongating behavior that [I already built out conditional code to avoid in the case of there being just 1 `KSelect`](https://github.com/learningequality/kolibri-design-system/pull/429/files#diff-11808f59cbc23e186d8a068a6b633df2d71c7319b7d93597a03eaedaa9597b51R281), but accidentally didn't account for how much more complex the relationship between the height properties would become once there were multiple `KSelect`s. i've been fiddling around with it frustratingly but haven't yet gotten it working as desired, and am not sure how much brain power is worth dedicating to that case.
![booooooo_AdobeExpress](https://user-images.githubusercontent.com/43046460/230692045-e2e1a530-a5d2-4507-a529-d173e147fd4e.gif)

Reply:
Hrm - interesting. The additional height only appears when the value is selected in the second one? Or it only happens when a value is selected in both?

Reply:
only appears once a value is selected in both -- the code i added to try to circumvent this problem is catching it the first time (whichever one is opened first), but because it was hacky it's not catching a second (and subsequent...) ones correctly. 

my understanding is that this happens in general because the existing function looks at the scrollheight of content to determine the height of the modal, and once the dropdown is opened it grabs its length as the newest scrollheight and makes adjustments. my hack was basically just skipping 1 round of this calculation if a KSelect was found in a modal, but it's not quite that straightforward.

it would be cool if we could just like...freeze the modal height forever if we knew there were `KSelect`s inside, except in theory a specific dropdown selection, depending on what it was, could result in a change to the rendered content so i figured it needed to remain dynamic

Reply:
so i think maybe the answer is patching up the updateContent calculation in a slightly different way than i have been? (/trying to work on making sure that scrollheight, etc in that function are targeting exactly what we expect and want them to be targeting)

Reply:
[resolved by changing the hacky-ness i was attempting -- originally was skipping a round of calculations if a select exists, now doing a different calculation entirely. gif of updated behavior below, with no more unexpected modal elongation after closing a second `KSelect` in a modal.]
![no-more-elongation](https://user-images.githubusercontent.com/43046460/230984386-074ce9c2-6a10-49e8-b236-38931e682413.gif)